mlflow/exceptions.py

```diff
@@ -67,6 +67,8 @@ class MlflowException(Exception):
     The error message associated with this exception may be exposed to clients in HTTP responses
     for debugging purposes. If the error text is sensitive, raise a generic `Exception` object
     instead.
+
+    Note: Instance of `MlflowException` is pickleable, see `__getstate__` and `__setstate__`
```

Comment:
Where is `__getstate__` and `__setstate__`?

Reply:
Removed. We only need to override `__reduce__`, if it has `__reduce__`, `__getstate__` and `__setstate__` is not used.